staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -176,12 +176,9 @@ func (f *FieldManager) Update(liveObj, newObj runtime.Object, manager string) (r
 // object and update the managed fields.
 func (f *FieldManager) Apply(liveObj runtime.Object, patch []byte, fieldManager string, force bool) (runtime.Object, error) {
 	// If the object doesn't have metadata, apply isn't allowed.
-	accessor, err := meta.Accessor(liveObj)
-	if err != nil {
+	if _, err := meta.Accessor(liveObj); err != nil {
 		return nil, fmt.Errorf("couldn't get accessor: %v", err)
 	}
-	missingManagedFields := (len(accessor.GetManagedFields()) == 0)
```

Comment:
dropping this isn't correct (it doesn't compile/pass unit tests). see if the hack script for creating the cherry pick comes up with a diff that matches the original PR better

Reply:
@liggitt The hack script modified much more than required for some reason https://github.com/kubernetes/kubernetes/pull/82581

Reply:
strange, it seemed to work for me... opened https://github.com/kubernetes/kubernetes/pull/82582